Content.Server/Radio/Components/RadioMicrophoneComponent.cs

```diff
@@ -34,6 +34,12 @@ public sealed partial class RadioMicrophoneComponent : Component
     [DataField("toggleOnInteract")]
     public bool ToggleOnInteract = true;
 
+    /// <summary>
+    // The radio frequency on which the message will be transmitted
+    /// </summary>
+    [DataField]
+    public int Frequency = 1459; // Common channel frequency
+
```

Comment:
This should just be set on mapinit to common's frequency.

Reply:
this was done, but the tests failed because component change during init.

Reply:
Update the yaml to do it.